package.json

```diff
@@ -26,6 +26,7 @@
     "debug": "^2.2.0",
     "lodash": "^4.17.4",
     "loopback-connector": "^4.0.0",
+    "loopback-connector-couchdb2": "https://github.com/strongloop/loopback-connector-couchdb2.git",
```

Comment:
I've released the beta for the CouchDB connector @ 0.9.0, so please change this accordingly!